client/src/redux/reducers/allocations.reducers.js

```diff
@@ -82,25 +82,33 @@ export function allocations(state = initialState, action) {
           ...action.payload.loading
         }
       };
-    case 'CLEAR_SEARCH_ERROR':
+    case 'SEARCH_INIT':
       return {
         ...state,
         search: {
           ...state.search,
-          error: false
+          results: [],
+          error: false,
+          loading: true
         }
       };
     case 'ADD_SEARCH_RESULTS':
       return {
         ...state,
-        search: { ...state.search, results: action.payload.data, error: false }
+        search: {
+          ...state.search,
+          results: action.payload.data,
+          error: false,
+          loading: false
+        }
       };
     case 'SEARCH_ERROR':
       return {
         ...state,
         search: {
           ...state.search,
-          error: true
+          error: true,
```

Comment:
Worth propagating an error code or message?

Reply:
dunno.  I think just showing "Something went wrong" is fine as a first pass. I think the only error would be backend is down or TAS is down. 🤔